build/release/release-procedure.adoc

```diff
@@ -61,47 +61,43 @@ postliminary actions.
-
-=== 3. Upgrade versions to the next snapshot
-*_This step is yet to be automated._*
-
-After the pipeline finishes, upgrade the version on the `main` branch of OptaPlanner to the next minor snapshot version,
-e.g. 8.0.0.Final -> 8.1.0-SNAPSHOT. Upgrade also the version of Drools by editing
-the value of the property `version.org.drools` in the `optaplanner-build-parent`.
-
-=== 4. Release OptaPlanner Quickstarts
-*_This step is yet to be automated._*
-
-. Create a release branch from the `development` branch.
-. Upgrade the OptaPlanner version in the `parent` section of the `pom.xml` and via the `version.org.optaplanner`
-property in all quickstarts, including Gradle build files, to the release version.
-. https://issues.redhat.com/browse/PLANNER-2230[Sync-up the stable branch from the release branch.]
-. Open a PR to the `stable` branch.
-. Create and push a tag for the release.
-. Upgrade the version to the next patch snapshot version on the release branch.
-.. Make sure the project builds.
-. Push the release branch.
-. Upgrade the version of the project to the next minor snapshot version via a PR to the `development` branch.
-
-== Kogito Release Pipeline
-
-The `kogito-release-pipeline` is a Jenkins pipeline that covers releases of the Kogito and the OptaPlanner projects
-together. As both projects consist of several Github repositories, the top-level `kogito-release-pipeline` orchestrates
-multiple pipelines that operate on the project, or the repository level.
-
-Please refer to the https://github.com/kiegroup/kogito-pipelines/blob/main/docs/nightly_and_release.md[Kogito release pipeline documentation]
-for more details.
-
-=== Actions covered by the Kogito release pipeline
-* Create releases branches and upgrade the projects to the release version.
-* Build the projects and deploy maven artefacts into a staging repository on JBoss Nexus.
-* Promote maven artifacts from the staging repository to a public one and later release this repository to Maven Central.
-* Upload the documentation and the distribution of all projects to the `filemgmt.jboss.org`.
-* Update links in the optaplanner-website to point to the released artifacts.
-* Upgrade version to the next bugfix snapshot on the release branch.
+find the version to be released and pick “Release” from the Actions. Create a next version for OptaPlanner and release the current version but WITHOUT
+release date.
+
+=== 2. Create a new branch for OptaPlanner release
+
+Run on Jenkins https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/optaplanner/job/0-prepare-release-branch[0-prepare-release-branch]
+to create a new branch where to release from. +
+Most links in this document take the 8.27.x branch as an example!
+
+=== 3. Update Drools
+
+Since OptaPlanner is not released anymore as a part of the Kogito release pipeline the Drools version is not upgraded automatically. This has to be done by running
```

Comment:
To keep it simple, let's focus on the current process without explaining how it used to be.
```suggestion
Run
```